### PR TITLE
Docs: Add 'tenant_id' for Loki clients (Auditing & Analytics)

### PR DESCRIPTION
**Pull request by joanlopez**
**What is this feature?**

Docs update according to https://github.com/grafana/grafana/pull/60866 and https://github.com/grafana/grafana-enterprise/pull/4276

**Why do we need this feature?**

Keep docs updated

**Who is this feature for?**

Auditing & Usage Insights users

**Which issue(s) does this PR fix?**:

N/A _(Discussed on Slack)_

**Special notes for your reviewer**:

Does that look enough? @mjseaman Any suggestion?

~~I'm wondering if I should create the corresponding [What's new?](https://github.com/grafana/grafana/tree/main/docs/sources/whatsnew) for `v9.4`?~~
~~Is it created manually just right before the stable release? It sounds suspicious that it doesn't exist yet.~~
Just discovered [whats-new-in-9.4](https://github.com/grafana/grafana/tree/docs/whats-new-in-9.4) branch, where that's already documented, so great 👌🏻 

PS: I added the `type/bug` label to make it "backportable" to `v9.4.x` (at the end I guess it could be considered a "docs bug" because I forgot to add those when I merged the code 😛)

Thanks!

### CODE REVIEW

**Comment by mjseaman** [comment]
@joanlopez my only small suggestion would be to link to the Loki docs that describe tenants, if those exist.

File: docs/sources/setup-grafana/configure-grafana/enterprise-configuration/index.md [hunk 1]
```diff
@@ -212,6 +216,10 @@ Set the URL for writing logs to Loki.
 
 If true, it establishes a secure connection to Loki. Defaults to true.
 
+### tenant_id
+
+Set the tenant id for Loki communications. Disabled by default. Mandatory to interact with Loki running in multi-tenant mode.
```
Comment (mjseaman) [inline docs/sources/setup-grafana/configure-grafana/enterprise-configuration/index.md:221]:
```suggestion
Set the tenant id for Loki communications. Disabled by default. Mandatory to interact with Loki running in [multi-tenant mode](https://grafana.com/docs/loki/latest/operations/multi-tenancy/).
```

File: docs/sources/setup-grafana/configure-grafana/enterprise-configuration/index.md [hunk 2]
```diff
@@ -102,6 +102,10 @@ Set the address for writing logs to Loki (format must be host:port).
 
 Decide whether or not to enable the TLS (Transport Layer Security) protocol when establishing the connection to Loki. Defaults to true.
 
+### tenant_id
+
+Set the tenant id for Loki communications. Disabled by default. Mandatory to interact with Loki running in multi-tenant mode.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/setup-grafana/configure-grafana/enterprise-configuration/index.md:107]: Links to docs in other repos follows this convention: [link text](/docs/repo/version/folder/file/).

I'll make that suggestion. Just wanted y'all to know.

File: docs/sources/setup-grafana/configure-security/audit-grafana.md [hunk 3]
```diff
@@ -420,6 +420,9 @@ type = grpc
 url = localhost:9095
 # Defaults to true. If true, it establishes a secure connection to Loki
 tls = true
+# Set the tenant id for Loki communications. Disabled by default.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/audit-grafana.md:423]:
```suggestion
# Set the tenant ID for Loki communications, which is disabled by default.
```

File: docs/sources/setup-grafana/configure-security/audit-grafana.md [hunk 4]
```diff
@@ -420,6 +420,9 @@ type = grpc
...
+# Mandatory to interact with Loki running in multi-tenant mode.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/audit-grafana.md:424]:
```suggestion
# The tenant ID is required to interact with Loki running in multi-tenant mode.
```

File: docs/sources/setup-grafana/configure-security/export-logs.md [hunk 5]
```diff
@@ -81,6 +81,9 @@ type = grpc
 url = localhost:9095
 # Defaults to true. If true, it establishes a secure connection to Loki
 tls = true
+# Set the tenant id for Loki communications. Disabled by default.
+# Mandatory to interact with Loki running in multi-tenant mode.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/export-logs.md:85]:
```suggestion
# The tenant ID is required to interact with Loki running in multi-tenant mode.
```

File: docs/sources/setup-grafana/configure-security/export-logs.md [hunk 6]
```diff
@@ -81,6 +81,9 @@ type = grpc
 url = localhost:9095
 # Defaults to true. If true, it establishes a secure connection to Loki
 tls = true
+# Set the tenant id for Loki communications. Disabled by default.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/export-logs.md:84]:
```suggestion
# Set the tenant ID for Loki communication, which is disabled by default.
```

**Review by chri2547 — CHANGES_REQUESTED** [review]
some minor suggestions. Thank you for the contribution!

**Comment by joanlopez** [comment]
> @joanlopez my only small suggestion would be to link to the Loki docs that describe tenants, if those exist.

> some minor suggestions. Thank you for the contribution!

Apologies for having this forgotten for weeks, finally applied the feedback, thanks for your suggestions & contributions! 

> Just wanted y'all to know.

Awesome, thanks for clarifying it! 

💟

**Review by chri2547 — APPROVED** [review]
Thank you for the contribution!

**Pull request merged**